getpaid/backends/transferuj/__init__.py

```diff
@@ -23,7 +23,13 @@ class PaymentProcessor(PaymentProcessorBase):
 
     _GATEWAY_URL = u'https://secure.transferuj.pl'
     _REQUEST_SIG_FIELDS = (u'id', u'kwota', u'crc',)
-    _ALLOWED_IP = (u'195.149.229.109', )
+    _ALLOWED_IP = (
+        u'195.149.229.109',
+        u'148.251.96.163',
```

Comment:
Jest to z "Dokumentacja techniczna Transferuj.pl" str 11
![image](https://cloud.githubusercontent.com/assets/1453207/26264070/b9f0c5de-3cdb-11e7-9c17-77bcf3dfac1c.png)

Wersja 1.7
![image](https://cloud.githubusercontent.com/assets/1453207/26264121/fbe484da-3cdb-11e7-8021-3bc5b66e00f6.png)